src/Extension/ValueResolver/StaticTextValueResolver.php

```diff
@@ -1,44 +0,0 @@
-<?php
```

Comment:
Please, mark as deprecated instead of removing it. We'll remove it in the next major release.